internal/cmd/org/export.go

```diff
@@ -298,12 +259,52 @@ var ExportCmd = &cobra.Command{
 				return err
 			}
 			if err = apiclient.WriteArrayByteArrayToFile(
-				environment+utils.DefaultFileSplitter+referencesFileName,
+				environment.Name+utils.DefaultFileSplitter+referencesFileName,
 				false,
 				referencesResponse); proceedOnError(err) != nil {
 				return err
 			}
 
+			if environment.Type != "BASE" {
+				clilog.Info.Printf("\tExporting KV Map names for environment %s...\n", environment)
+				if listKVMBytes, err = kvm.List(""); err != nil {
```

Comment:
I think we need to use `proceedOnError` here.
BASE doesn't support KVMs or Flow Hooks